1 The Physical/Guess Point Level 1 (Stateless)/1 - Red-Green-Refactor Practice/1.1 - Fizzbuzz/src/fizzbuzz.ts

```diff
@@ -0,0 +1,22 @@
+export default class FizzBuzz {
+  get(input: number): string {
+    let s = "";
```

Comment:
naming should be more explicit like result
also this is introducing state (do we really need it ?)